dali/operators/python_function/jax_function.h

```diff
@@ -70,7 +70,7 @@ DLMTensorPtr SetupTensorResource(std::unique_ptr<DLDaliTensorResource<Backend>>
 }
 
 template <typename Backend>
-DLMTensorPtr AsDLTensor(Tensor<Backend> tensor) {
+DLMTensorPtr AsDLTensor(Tensor<Backend> &&tensor) {
```

Comment:
Usages of this would also need to be `std::move(tensor)`, is that correct?

Reply:
Yes, and they are